packages/e2e/test/predefined-data/addresses/testnet.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
As far as I remember `testnet` no longer exists.
Maybe this can be deleted-

Reply:
This was associated with `predprod` since you said will extract a set of addresses for it as well.
I can remove it or rename it to `preprod` ? @iccicci